examples/large_models/segment_anything_fast/model-config.yaml

```diff
@@ -0,0 +1,6 @@
+responseTimeout: 300
+handler:
+    profile: true
+    model_type: "vit_h"
+    sam_checkpoint: "/home/ubuntu/serve/examples/large_models/segment_anything_fast/sam_vit_h_4b8939.pth"
```

Comment:
Can we use a relative path here or not have this hard coded?

Reply:
for torch compiler, do we need to pass any other additional flags in the config file?

Reply:
We have only 1 known way to do this https://github.com/pytorch/serve/tree/master/examples/large_models/Huggingface_accelerate/llama2

However, this was inconvenient for repeated experiments. We have to move the weights before creating mar file or redownload them.

The current approach is simple.

Reply:
No flags needed for torch.compile. Everything is done in SAM Fast